samples/ingestion/ingestion-client/FetchTranscription/FetchTranscriptionEnvironmentVariables.cs

```diff
@@ -40,12 +40,16 @@ public static class FetchTranscriptionEnvironmentVariables
 
         public static readonly string FetchTranscriptionServiceBusConnectionString = Environment.GetEnvironmentVariable(nameof(FetchTranscriptionServiceBusConnectionString), EnvironmentVariableTarget.Process);
 
+        public static readonly string FetchTranscriptionServiceBusQueueName = Environment.GetEnvironmentVariable(nameof(FetchTranscriptionServiceBusQueueName), EnvironmentVariableTarget.Process);
+
         public static readonly string HtmlResultOutputContainer = Environment.GetEnvironmentVariable(nameof(HtmlResultOutputContainer), EnvironmentVariableTarget.Process);
 
         public static readonly string JsonResultOutputContainer = Environment.GetEnvironmentVariable(nameof(JsonResultOutputContainer), EnvironmentVariableTarget.Process);
 
         public static readonly string StartTranscriptionServiceBusConnectionString = Environment.GetEnvironmentVariable(nameof(StartTranscriptionServiceBusConnectionString), EnvironmentVariableTarget.Process);
 
+        public static readonly string StartTranscriptionServiceBusQueueName = Environment.GetEnvironmentVariable(nameof(StartTranscriptionServiceBusQueueName), EnvironmentVariableTarget.Process);
```

Comment:
Why did these need to be added?

Reply:
Ah, I guess the names were previously picked up from the connection strings. We should be able to just use the ConnectionStringProperties type to parse out the queue name.
https://docs.microsoft.com/en-us/dotnet/api/azure.messaging.servicebus.servicebusconnectionstringproperties.entitypath?view=azure-dotnet#Azure_Messaging_ServiceBus_ServiceBusConnectionStringProperties_EntityPath